roles-maintainer.adoc

```diff
@@ -23,7 +23,7 @@ The growing Bisq Network requires a system of roles in order to distribute respo
 
 == Privileges
 
-Admin access to the roles repository.
+Admin access to the roles repository at https://lists.bisq.network/roles.
```

Comment:
This should be https://github.com/bisq-network/roles.

Reply:
yeah, got it.